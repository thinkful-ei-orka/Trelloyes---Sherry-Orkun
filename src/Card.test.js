import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';

import Card from './Card';

 describe('<Card />', () => {
   it('renders without crashing', () => {
     const div = document.createElement('div');

     ReactDOM.render(
       <Card
          title="To-Do"
          content="sleep"
        />,
        div
      );

      ReactDOM.unmountComponentAtNode(div);
   })

   it('renders this UI as expcected', () => {
     const tree = renderer.create(<Card
      title="To-Do"
      content="sleep"
      />).toJSON();
      
      expect(tree).toMatchSnapshot();
   });
 });