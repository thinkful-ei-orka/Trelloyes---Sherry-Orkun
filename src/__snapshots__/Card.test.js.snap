// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Card /> renders this UI as expcected 1`] = `
<div
  className="Card"
>
  <button
    type="button"
  >
    delete
  </button>
  <h3>
    To-Do
  </h3>
  <p>
    sleep
  </p>
</div>
`;
